README.CentOS7.md

```diff
@@ -43,7 +43,7 @@ MySQL is not recommended/supported on CentOS, see [paragraph 2 here](https://www
 *Note:* LORIS developers (those NOT working with a .zip release codebase) should skip steps relating to hosting their database locally. Contact your system administrator for database credentials.
 
 ```bash
-sudo yum install MariaDB-client MariaDB-server
+sudo yum install mariadb-client mariadb-server
```

Comment:
Which version of CentOS are you using?
The current letter case is correct for CentOS7 (this is CentOS7.md)
I would accept a note that indicate that for CentOS8, the package names are lowercase.

Reply:
@PapillonMcGill I'm using CentOS 7

Reply:
correct. MariaDB is on mariadb repo not base centos repo